src/App.jsx

```diff
@@ -14,45 +17,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const buttons = [
+  {
+    name: 'Sort alphabetically',
+    class: 'is-info',
+  },
+  {
+    name: 'Sort by length',
+    class: 'is-success',
+  },
+];
+
+const sortList = (goods, buttonClass, isReversed) => {
+  const list = [...goods];
+  const sortLength = (good1, good2) => good1.length - good2.length;
+
+  if (buttonClass === 'is-info') {
+    return isReversed ? list.sort().reverse() : list.sort();
```

Comment:
Just reverse the list at the end of the function if it's reversed to not repeat the reverse logic